panel/auth.py

```diff
@@ -1078,6 +984,60 @@ def login_handler(self):
         handler._login_endpoint = self._login_endpoint
         return handler
 
+    def _remove_user(self, session_context):
+        user_cookie = session_context.request.cookies.get('user')
+        user = decode_signed_value(config.cookie_secret, 'user', user_cookie).decode('utf-8')
+        state._active_users[user] -= 1
+        if not state._active_users[user]:
+            del state._active_users[user]
+            if user in state._oauth_user_overrides:
+                del state._oauth_user_overrides[user]
+
+    def _schedule_refresh(self, expiry_ts, user, refresh_token, application, request):
+        if not state._active_users.get(user):
+            return
+        now_ts = dt.datetime.now(dt.timezone.utc).timestamp()
+        expiry_seconds = expiry_ts - now_ts - 10
+        log.debug("%s scheduling token refresh in %d seconds", type(self).__name__, expiry_seconds)
+        expiry_date = dt.datetime.now() + dt.timedelta(seconds=expiry_seconds)
```

Comment:
Shouldn't this also use `dt.timezone.utc`?

Reply:
No, `schedule_task` works in the local timezone.

Reply:
This would be nice to have as a small comment, I think.